pkg/commands/login.go

```diff
@@ -39,14 +100,30 @@ func RunLogin(ctx CommandContext, args LoginArgs) error {
 		Token:        strings.Trim(args.Token, "\n"),
 	}
 
-	api, err := api.NewScalewayAPI(cfg.ComputeAPI, cfg.AccountAPI, cfg.Organization, cfg.Token)
+	apiConnection, err := api.NewScalewayAPI(cfg.ComputeAPI, cfg.AccountAPI, cfg.Organization, cfg.Token)
 	if err != nil {
 		return fmt.Errorf("Unable to create ScalewayAPI: %s", err)
 	}
-	err = api.CheckCredentials()
+	err = apiConnection.CheckCredentials()
 	if err != nil {
 		return fmt.Errorf("Unable to contact ScalewayAPI: %s", err)
 	}
+	if args.IndexSSH != "" {
+		uuid, err := apiConnection.GetUUID()
+		if err != nil {
+			return fmt.Errorf("Unable to contact ScalewayAPI: %s", err)
```

Comment:
Not 100% sure, but I think we should consider than `login` works even if the SSH key failed to upload, so maybe we can just print the error with `logrus.Errorf`, or we can still return the error but maybe we can call  `cfg.Save()` just after the successful `apiConnection.CheckCredentials()`